src/main/java/com/synopsys/integration/rest/client/IntHttpClient.java

```diff
@@ -279,14 +293,12 @@ private void addBuilderCredentialsProvider() {
 
     private void addBuilderSSLContext() {
         try {
-            SSLContext sslContext;
             HostnameVerifier hostnameVerifier;
             if (alwaysTrustServerCertificate) {
                 logger.error("Automatically trusting server certificates - not recommended for production use.");
                 sslContext = SSLContextBuilder.create().loadTrustMaterial(new TrustAllStrategy()).build();
                 hostnameVerifier = new NoopHostnameVerifier();
             } else {
-                sslContext = SSLContexts.createDefault();
```

Comment:
What is the goal of passing in the sslContext? When the `alwaysTrustServerCertificate` it seems like that would ruin the purpose of the change you have made. It seems like when `alwaysTrustServerCertificate` is true you would need to take the sslContext provided and use it to create a new sslContext that will have the `TrustAllStrategy()`

Reply:
We're all over this, thanks James!